src/strategies/ens-all-club-digits/index.ts

```diff
@@ -0,0 +1,76 @@
+import { subgraphRequest } from '../../utils';
+import { getAddress } from '@ethersproject/address';
+
+const ENS_SUBGRAPH_URL = {
+  '1': 'https://api.thegraph.com/subgraphs/name/ensdomains/ens',
+  '3': 'https://api.thegraph.com/subgraphs/name/ensdomains/ensropsten',
+  '4': 'https://api.thegraph.com/subgraphs/name/ensdomains/ensrinkeby',
+  '5': 'https://api.thegraph.com/subgraphs/name/ensdomains/ensgoerli'
+};
+
+export const author = '3757';
+export const version = '0.1.1';
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const { numberOfDigits } = options;
+  // 999 Club = 3 , 10k club = 4, 100k club = 5
+  const NAMES_CLUB = [...Array(Math.pow(10, numberOfDigits)).keys()]
+    .map((i) => -1 + i + 1)
+    .map((x) => x.toString().padStart(numberOfDigits, '0'));
```

Comment:
I feel like someone could pass a huge number and it can cause memory issues here, maybe there should be a limit for this

Reply:
Thanks for pointing this out. I have tested this, and up to 7 digits it worked like a charm on my machine. At 8 digits it kind of got "stuck", so I made it support 3 digits (lowest number ENS has) up to 7 digits. Any other input will not work into this field

We principally need this snapshot for 3,4,5 digit numbers so it should be all good

Reply:
@onigiri-x Can make it max to 5? I tried changing it to 7 and the tests are failing in GitHub actions. 

Or you see any alternate way to check with out adding all possible values to an array? we can't check response directly?